setup.py

```diff
@@ -31,7 +31,7 @@
                 if package.startswith('gym')],
       zip_safe=False,
       install_requires=[
-          'scipy', 'numpy>=1.10.4', 'six', 'pyglet>=1.2.0',
+          'scipy', 'numpy>=1.10.4', 'six', 'pyglet>=1.2.0', 'cloudpickle'
```

Comment:
This should specify `cloudpickle~=<version number>` to make sure it doesn't randomly break later.